packages/app/src/components/Text/NumericValue.tsx

```diff
@@ -11,11 +11,12 @@ type NumericValueProps = BodyProps & {
 	colored?: boolean
 	options?: FormatNumberOptions
 	suffix?: string
+	span?: 'span' | 'p'
```

Comment:
It looks like an artifact because the component uses `type`. Is this the right `prop` exactly?

Reply:
yea good catch, was meant to be `type`

Reply:
was left in by mistake, removed